release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2704,6 +2704,78 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+[id="ocp-4-15-12"]
+=== RHSA-2024:2664 - {product-title} 4.15.12 bug fix and security update
+
+Issued: 2024-05-09
+
+{product-title} release 4.15.12, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:2664[RHSA-2024:2664] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:2669[RHSA-2024:2669] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.15.12 --pullspecs
+----
+
+[id="ocp-4-15-12-enhancements"]
+==== Enhancements
+
+The following enhancements are included in this z-stream release:
+
+[id="ocp-4-15-12-apiversion-upgrade-v1beta1"]
+===== API version for the ClusterTriggerBinding, TriggerTemplate, and EventListener CRDs upgraded from v1alpha1 to v1beta1
+
+* Previously, the API version for the ClusterTriggerBinding, TriggerTemplate, and EventListener CRDs was v1alpha1. With this release, the API version is upgraded to v1beta1 so that the pipelines plugin supports the latest Pipeline Trigger API version for the ClusterTriggerBinding, TriggerTemplate, and EventListener CRDs. (link:https://issues.redhat.com/browse/OCPBUGS-31445[*OCPBUGS-31445*])
```

Comment:
```suggestion
* Previously, the API version for the `ClusterTriggerBinding`, `TriggerTemplate`, and `EventListener` CRDs was `v1alpha1`. With this release, the API version is upgraded to `v1beta1` so that the pipelines plugin supports the latest Pipeline Trigger API version for the `ClusterTriggerBinding`, `TriggerTemplate`, and `EventListener` CRDs. (link:https://issues.redhat.com/browse/OCPBUGS-31445[*OCPBUGS-31445*])
```

Reply:
good call!